echemdb/data/cv/entry.py

```diff
@@ -175,29 +175,29 @@ def format_title(self, e, which_field, as_sentence=True):
 
     def x(self):
         r"""
-        Return the name of the variable on the x-axis, i.e., `"U"`.
-
-        TODO: Adapt along with https://github.com/echemdb/svgdigitizer/issues/106.
+        Return the name of the variable on the x-axis, i.e., `"E"`.
 
         EXAMPLES::
 
             >>> entry = Entry.create_examples()[0]
             >>> entry.x()
-            'U'
+            'E'
 
         """
         from astropy import units as u
-        if u.Unit(self.figure_description.potential_scale.unit).is_equivalent('V'):
-            return 'U'
+
+        if not self.data_description.axes:
+            raise ValueError(f"No axes were specified for the dataset.")
+        if 'E' in self.data_description.axes._descriptor.keys():
+            assert u.Unit(self.data_description.axes.E.unit).is_equivalent('V'), f"The variable on the x-axis is not equivalent to 'V'."
```

Comment:
```suggestion
            assert u.Unit(self.data_description.axes.E.unit).is_equivalent('V'), f"The variable on the x-axis is not convertible to 'V'."
```

I don't think people understand what they mean with "equivalent" here.

Reply:
agreed